src/sage/algebras/hecke_algebras/cubic_hecke_base_ring.py

```diff
@@ -1353,7 +1353,7 @@ def specialize_homfly(self):
             True
         """
         if not self._is_markov_trace_version():
-            raise ValueError('Functionality available for Markov trace version, only')
+            raise ValueError('functionality available for Markov trace version, only')
```

Comment:
```suggestion
            raise ValueError('functionality only available for Markov trace version')
```
I think this English is better.